org.civicrm.bootstrapcivicrm/scss/bootstrap/bootstrap.scss

```diff
@@ -5,14 +5,14 @@
 @import "mixins/*";
 
 #{$bootstrap-namespace} {
-    @import "vendor/text-angular/text-angular";
-    @import "vendor/ui-bootstrap/ui-bootstrap";
-    @import "vendor/ui-select/ui-select";
+  @import "vendor/ui-bootstrap/ui-bootstrap";
+  @import "vendor/ui-select/ui-select";
 
-    @import "overrides/style/*";
-    @import "components/*";
+  @import "overrides/style/*";
+  @import "components/*";
 }
 
+@import "vendor/text-angular/text-angular";
```

Comment:
Why did you move it outside the namespace? Are you sure this does not affect any other pages where text-angular is used?